### ISSUE TITLE
stack build fails building HDBC-postgresql package

### ISSUE THREAD

**Issue by develop7** [FlogFr/FlashCard#3]
Repro: clone project, run `stack build` or `stack build HDBC-postgresql`

[HDBC-postgresql-2.3.2.5.log](https://github.com/FlogFr/FlashCard/files/2650581/HDBC-postgresql-2.3.2.5.log)

**Comment by FlogFr** [comment]
Thanks for the issue.

This project was mainly to discover Haskell and ELM and as I have more experience now with Haskell, I will also refactor a bit :)

**Comment by develop7** [comment]
Yeah, it's just I wonder how did you even ran this if the straight-up build fails?

**Comment by develop7** [comment]
'Cause the issue appears to be in the upstream library.

**Comment by FlogFr** [comment]
@develop7 I finally refactored all the application in this PR: https://github.com/FlogFr/FlashCard/pull/4
I will deploy it soon.
I'm not using the library HDBC anymore. I prefered the raw bindings of the LibPQ with some typeclass :)
Let me know if you can build the project from the new master branch.
Thanks

**Comment by develop7** [comment]
@FlogFr that's great! thank you! Still, did the code ever worked before this PR?

**Comment by FlogFr** [comment]
@develop7 yes, it worked. But I don't know what's the good way to include `HDBC-postgresql` to a haskell project.
